bugbug/similarity.py

```diff
@@ -390,3 +391,39 @@ def get_distance(self, query1, query2):
         wmd = self.wmdistance(words1, words2, all_distances)
 
         return wmd
+
+
+class SoftCosSimilarity(BaseSimilarity):
```

Comment:
Could you investigate having a Word2VecSimilarity base class, and derive WMD and SoftCos from that?